composer.json

```diff
@@ -21,8 +21,7 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ^8.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "^7.3 || ^8.0"
```

Comment:
IMO `^7.3 || ~8.0 || ~8.1` at this point...

Reply:
But with the patch `0` as othetwise, tilde works the same way as caret.

Reply:
should be:

```
^7.3 || ~8.0.0 || ~8.1.0
```

Reply:
Yes - have this poised to push - just waiting on two more clarifications on other items before I do so.